src/main/java/org/cactoos/scalar/SumOfScalar.java

```diff
@@ -39,6 +39,10 @@
  * <p>Note this class is for internal usage only
  *
  * @since 0.30
+ * @todo #1266:30min This class uses Java8 Stream API. Refactor the
```

Comment:
@fanifieiev Looks like this todo is not related to the original issue. You should file a new bug for it